backend/wrapped/serializers.py

```diff
@@ -0,0 +1,105 @@
+from rest_framework import serializers
+from .models import Page, IndividualStat, GlobalStat, IndividualStatThrough, GlobalStatThrough, Semester, User
+
+class IndividualStatSerializer(serializers.ModelSerializer):
+    key = serializers.StringRelatedField()
```

Comment:
Let's not use StringRelatedField because it relies on the `__str__` which is ususally used for testing. Can you find a better one to use?